### DISCUSSION TITLE
Making a clear all button

### DISCUSSION THREAD

**Discussion by tomjthomas** [prevwong/craft.js#238] [Q&A]
How can I implement a button that would delete all the nodes in the editing area?

I was planning on going through all the children of the edit area and deleting them one by one but I can only access the children from the editing area component (or that's what I understood). I thought of a workaround by setting the editing area as the selected node and then creating a related component.

Is there a better way to do this?

**Comment by cederron** [comment]
I'm doing
`    actions.deserialize('{"ROOT":{"type":{"resolvedName":"Container"},"isCanvas": true,"props":{},"displayName":"Project","custom":{},"hidden":false,"nodes":[],"linkedNodes":{}}}');
`
